code/parse/sexp.cpp

```diff
@@ -10705,15 +10709,15 @@ int eval_for_players(int arg_handler_node, int condition_node, bool just_count =
 			if (just_count)
 				num_valid_arguments++;
 			else
-				argument_vector.emplace_back(vm_strdup(Player_ship->ship_name), -1);
+				argument_vector.emplace_back(Player_ship->ship_name, -1);
```

Comment:
vm_strdup is removed here. What effect does that have, exactly?

Reply:
Same as above